checker/build.xml

```diff
@@ -787,7 +787,7 @@
     </target>
 
     <target name="nullness-safedefaultsbytecode-tests" depends="jar,build-tests"
-            description="Run tests for the Nullness Checker using -AsafeDefaultsForUnannotatedBytecode">
+            description="Run tests for the Nullness Checker using -AuseDefaultsForUncheckedCode=-source,bytecode">
         <antcall target="-run-tests">
             <param name="param" value="tests.NullnessSafeDefaultsBytecodeTest"/>
         </antcall>
```

Comment:
There are two descriptions below that mention -AuseSafeDefaultsForUnannotatedSourceCode